lib/assets/stylesheets/modules/_technical-documentation.scss

```diff
@@ -42,65 +46,65 @@
 
   // Headings
   h1 {
```

Comment:
Can we use the new typography classes from Frontend? We do this in the Design System [like this](https://github.com/alphagov/govuk-design-system/blob/master/src/stylesheets/main.scss#L302-L351)